server/base/src/main/java/org/apache/accumulo/server/master/balancer/TableLoadBalancer.java

```diff
@@ -47,8 +47,8 @@
 
   private TabletBalancer constructNewBalancerForTable(String clazzName, String table) throws Exception {
     String context = null;
-    if (null != configuration)
-      context = configuration.getTableConfiguration(table).get(Property.TABLE_CLASSPATH);
+    if (null != tabletBalancerEnvironment.getConfiguration())
```

Comment:
If it's null, I would consider that a bug, probably wouldn't want to hide it. Not sure why it checked for null before.